src/utils/_colors.scss

```diff
@@ -12,25 +12,25 @@
 // $c-primary-dark: color-variation($c-primary, -2) -> 2 steps darker
 // $c-primary-darkest: color-variation($c-primary, -4) -> 4 steps darker
```

Comment:
What is the problem you were facing with current implementation ??
AFAIK step variations are not supposed to be overwitten:
https://paper.dropbox.com/doc/Basic-Styles--AJrkoC13T7nSRdECBbSq37VvAg-0ko2wfsTt1JOQHV6G8cwo#:uid=016947049972010127346188&h2=Color-Palette

Reply:
Transfer this question to UX. I block the PR until I have an answer.